include/RAJA/index/IndexSetUtils.hpp

```diff
@@ -49,11 +49,8 @@ RAJA_INLINE void getIndices(CONTAINER_T& con,
                             const TypedIndexSet<SEG_TYPES...>& iset)
 {
   CONTAINER_T tcon;
-  forall<ExecPolicy<seq_segit, seq_exec> >(iset,
-    [&](typename CONTAINER_T::value_type idx) {
-      tcon.push_back(idx);
-    }
-  );
+  forall<ExecPolicy<seq_segit, seq_exec>>(
+      iset, [&](typename CONTAINER_T::value_type idx) { tcon.push_back(idx); });
```

Comment:
I think it's better to start lambda expressions on a new line. Also, why did it not format the curly braces on new lines like other code sections?

Reply:
I have it set to allow single line lambdas